packages/webpack-cli/src/webpack-cli.ts

```diff
@@ -2320,14 +2487,14 @@ class WebpackCLI {
       options.watch = true;
     }
 
-    compiler = await this.createCompiler(options, callback);
+    compiler = await this.createCompiler(options as WebpackDevServerOptions, callback);
 
     if (!compiler) {
       return;
     }
 
-    const isWatch = (compiler) =>
-      compiler.compilers
+    const isWatch = (compiler: WebpackCompiler): undefined | boolean =>
```

Comment:
Same here, only `boolean` should be returned, technically `undefined` is `false`, but it is not good practice

Reply:
Thanks, I'll just cast both method using `Boolean()`